release/release.js

```diff
@@ -75,6 +75,7 @@ const checkoutAndPullLatestCode = async () => {
   stepTitle('👀 Checking out the latest branch...')
   const branchToCheckout = config.data.isFirstReleaseIteration ? 'development' : `release/${VERSION}`
   console.log(`Great, checking out ${chalk.magenta(branchToCheckout)}`)
+  console.log()
```

Comment:
Empty `console.log()`?

Reply:
Doing this to add the line break space instead of relying on the newline character to try to solve the stray "n" characters we were seeing at the end of some of the output lines.

Reply:
👌